README.md

```diff
@@ -1,104 +1,128 @@
+| `turbineheatprocessor` | scenario | A Flink streaming application for processing temperature data from a Pravega stream. Complements the `turbineheatsensor` app. The application computes a daily summary of the temperature range observed on that day by each sensor. | [Java](https://github.com/pravega/pravega-samples/tree/master/flink-examples/src/main/java/io/pravega/examples/flink/iot), [Scala](https://github.com/pravega/pravega-samples/tree/master/flink-examples/src/main/scala/io/pravega/examples/flink/iot)
+| `anomaly-detection` | scenario | A Flink streaming application for detecting anomalous input patterns using a finite-state machine. | [Java](https://github.com/pravega/pravega-samples/tree/master/anomaly-detection)
+
+# Build Instructions
+
+## Pre-requisites
+
+* Java 8
+
+* Maven 2
+
+## Pravega Build Instructions (Optional)
+
+If you have downloaded a nightly build of Pravega, you may need to generate the latest 
+Pravega `jar` files and publish them to your local Maven repository. For release builds of 
+Pravega, the artifacts will already be in Maven Central and you will not need to run this step.
+
+To install the Pravega libraries to your local Maven repository:
 
-Install the Pravega client libraries to your local Maven repository:
 ```
 $ git clone https://github.com/pravega/pravega.git
-$./gradlew install
+$ cd pravega
+$ ./gradlew install
 ```
 
-### Building the Flink Connector
+The above command should generate the required `jar` files into your local Maven repository.
+For more information, please visit [Pravega Git repository](https://github.com/pravega/pravega). 
 
-Optional: This step is required only if you want to use a different version
-of Pravega than is published to maven central.
+> Hint: If you use a different version of Pravega, please check the gradle.properties file.
 
-Install the shaded Flink Connector library to your local Maven repository:
-```
-$ git clone https://github.com/pravega/flink-connectors.git
-$./gradlew install
-```
+## Flink Connector Build Instructions
 
-### Building the Samples
-Use the built-in gradle wrapper to build the samples.
-```
-$ ./gradlew build
-...
-BUILD SUCCESSFUL
-```
+To execute Flink connector examples, follow the below steps to build and publish artifacts from 
+source to local Maven repository:
 
-### Distributing (Flink Samples)
-#### Assemble
-Use gradle to assemble a distribution folder containing the Flink programs as a ready-to-deploy uber-jar called `pravega-flink-examples-0.1.0-SNAPSHOT-all.jar`.
 ```
-$ ./gradlew installDist
-...
-$ ls -R flink-examples/build/install/pravega-flink-examples
-bin	lib
+$ git clone --recursive https://github.com/pravega/flink-connectors.git
+$ cd flink-connectors
+$ ./gradlew clean install
+```
 
-flink-examples/build/install/pravega-flink-examples/bin:
-run-example
+For more information, please visit [Flink connector Git repository](https://github.com/pravega/flink-connectors). 
```

Comment:
please visit Flink Connectors should be enough, with the link pointing to GitHub repo.

Reply:
Done.